flow-typed/@react-navigation/bottom-tabs_v5.x.x.js

```diff
@@ -245,9 +239,7 @@ declare module '@react-navigation/bottom-tabs' {
     label?: string,
     ...
   }>;
-  declare type AccessibilityActionEvent = SyntheticEvent<
-    $ReadOnly<{actionName: string, ...}>,
-  >;
+  declare type AccessibilityActionEvent = SyntheticEvent<$ReadOnly<{ actionName: string, ... }>>;
```

Comment:
react-navigation libdef [nfc]: Format files.

I think it'd be better to leave them in the format we found them in. The reason is that ideally we'd be contributing these fixes back upstream to the flow-typed repo, and preserving the format will help keep it smooth for us to do so.

So probably that means telling Prettier and ESLint to ignore these files.